### PR TITLE
Adds support for hex values, rgba strings, malformed values with fail…

### PR DESCRIPTION
**Pull request by Lucassifoni**
See issue #418 .
I added some test cases, but the coverage significantly dropped.
Still, function types match and should provide a safety net.
Have a nice day !

### CODE REVIEW

**Comment by Lucassifoni** [comment]
It seems the travis integration consistently fails with Node < 6, maybe specifying a node minimum version would benefit the project too !

**Comment by peterramsing** [comment]
@Lucassifoni This looks awesome. It looks like we need to get some tests in place to validate the fixes. Let me know if you need any help writing those tests. I'd love to help!

File: lib/lost-utility.js [hunk 1]
```diff
@@ -1,9 +1,96 @@
...
+  if (testRgb.test(string)) {
+    return safeRgbToRgb(string);
+  }
+  if (testHex.test(string)) {
+    return safeHexToRgb(string);
+  }
+  return [0, 0, 255];
+}
+
+var testRgb = new RegExp(/rgb/);
```
Comment (peterramsing) [inline lib/lost-utility.js]: It sounds silly, but could this either be at the top or the bottom of the file? In the middle for a "const" feels a bit odd.

**Review by peterramsing — CHANGES_REQUESTED** [review]
This looks like some pretty swell code! It needs some test coverage and with that, I'd say it's a sweet addition. Thank you!

**Comment by peterramsing** [comment]
It also looks like a lint failed here: https://travis-ci.org/peterramsing/lost/jobs/427015147#L1112.

I've been looking into the lint config. I'm hesitant to turn this off but it has been falsely failing. Feel free to add `//eslint-ignore-next-line` if you feel it's needed.

**Comment by Lucassifoni** [comment]
@peterramsing Hi Peter, thanks for your feedback.
I'll update this with tests and reorganize the code this morning !

File: lib/lost-utility.js (hunk 1 above)
Comment (Lucassifoni) [reply lib/lost-utility.js]: I updated organization and added tests, it should be green for both tests & lints now.
Thanks for your guidance.

**Comment by peterramsing** [comment]
This is lovely. I'm inspired by your code style. I'm gonna do some refactors of my own. Thanks for some awesome code!

**Comment by peterramsing** [comment]
...as for the package-lock? meh... I'm not sure why that's there but I'm doing some refactors that will blow that away so I'm not too worried about that.

**Review by peterramsing — APPROVED** [review]

**Pull request merged**